code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -0,0 +1,885 @@
+	)
+	var/chosen = pick(SSeconomy.supply_packs - ignored_supply_pack_types)
+	var/datum/supply_packs/the_pack = new chosen()
+	var/spawn_location = get_turf(target)
+	var/obj/structure/closet/crate/crate = the_pack.create_package(spawn_location)
+	crate.name = "magic [crate.name]"
+	qdel(the_pack)
+
+/datum/tarot/reversed/the_hermit
+	name = "IX - The Hermit?"
+	desc = "May you see the value of all things in life."
+	card_icon = "the_hermit?"
+
+/datum/tarot/reversed/the_hermit/activate(mob/living/target) //Someone can improve this in the future (hopefully comment will not be here in 10 years.)
+	for(var/obj/item/I in view(7, target))
+		if(istype(I, /obj/item/gun))
+			new /obj/item/stack/spacecash/c200(get_turf(I))
+			qdel(I)
+			continue
+		if(istype(I, /obj/item/grenade))
+			new /obj/item/stack/spacecash/c50(get_turf(I))
+			qdel(I)
+		if(istype(I, /obj/item/clothing/suit/armor))
+			new /obj/item/stack/spacecash/c100(get_turf(I))
+			qdel(I)
+		if(istype(I, /obj/item/melee/baton))
+			new /obj/item/stack/spacecash/c100(get_turf(I))
+			qdel(I)
+
+/datum/tarot/reversed/wheel_of_fortune
+	name = "X - Wheel of Fortune?"
+	desc = "Throw the dice of fate."
+	card_icon = "wheel_of_fortune?"
+
+/datum/tarot/reversed/wheel_of_fortune/activate(mob/living/target)
+	var/obj/item/dice/d20/fate/one_use/gonna_roll_a_one = new /obj/item/dice/d20/fate/one_use(get_turf(target))
+	gonna_roll_a_one.diceroll(target)
+
+/datum/tarot/reversed/strength
+	name = "XI - Strength?"
+	desc = "May you break their resolve."
+	card_icon = "strength?"
+
+/datum/tarot/reversed/strength/activate(mob/living/target)
+	for(var/mob/living/M in oview(9, target))
+		M.Hallucinate(2 MINUTES)
+		new /obj/effect/hallucination/delusion(get_turf(M), M)
+		M.adjustBrainLoss(30)
+
+/datum/tarot/reversed/the_hanged_man
+	name = "XII - The Hanged Man?"
+	desc = "May your greed know no bounds."
+	card_icon = "the_hanged_man?"
+
+/datum/tarot/reversed/the_hanged_man/activate(mob/living/target)
+	var/obj/structure/cursed_slot_machine/pull_the_lever_kronk = new /obj/structure/cursed_slot_machine(get_turf(target))
+	if(ishuman(target))
+		var/mob/living/carbon/human/WRONG_LEVER = target
+		pull_the_lever_kronk.attack_hand(WRONG_LEVER)
+
+/datum/tarot/reversed/death
+	name = "XIII - Death?"
+	desc = "May life spring forth from the fallen."
+	card_icon = "death?"
+
+/datum/tarot/reversed/death/activate(mob/living/target)
+	new /obj/structure/constructshell(get_turf(target))
+	new /obj/item/soulstone/anybody(get_turf(target))
+
+/datum/tarot/reversed/temperance
+	name = "XIV - Temperance?"
+	desc = "May your hunger be satiated."
+	card_icon = "temperance?"
+
+/datum/tarot/reversed/temperance/activate(mob/living/target)
+	if(!ishuman(target))
+		return
+	var/mob/living/carbon/human/H = target
+	for(var/i in 1 to 5)
+		var/datum/reagents/R = new /datum/reagents(10)
+		R.add_reagent(get_unrestricted_random_reagent_id(), 10)
+		R.reaction(H, REAGENT_INGEST)
+		R.trans_to(H, 10)
+		i++
```

Comment:
```suggestion

```
The for loop already increments i, so this'd make it only give 3 reagents

Reply:
oh whoops I must have doubled up, thank you